src/review/commands/share/hookHandlers/pushHookHandler.ts

```diff
@@ -29,7 +29,7 @@ export async function pushHookHandler(
     return;
   }
 
-  const branchName = ref.split('/').pop() as string;
+  const branchName = ref.split('/').slice(2).join('/') as string;
```

Comment:
Is it always two words before having branch name? I suppose here we are assuming the ref will always start with something like `refs/branch`, right?